analyzers/dataframe/src/myUtils.cc

```diff
@@ -184,10 +210,10 @@ ROOT::VecOps::RVec<float> get_Vertex_d2PVError(ROOT::VecOps::RVec<VertexingUtils
   ROOT::VecOps::RVec<float> result;
   VertexingUtils::FCCAnalysesVertex PV;
   for (auto &p:vertex)
-    if (p.vertex.primary>0) PV=p;
+    if (isPrimaryVtx(p.vertex)) PV=p;
 
   for (auto &p:vertex){
-    if (p.vertex.primary>0) result.push_back(0);
+    if (isPrimaryVtx(p.vertex)) result.push_back(0);
```

Comment:
This is the same as above for `get_Vertex_d2PV`, so for consistency I went with the same implementation here.